Client/src/Hooks/LocalBackedState.ts

```diff
@@ -0,0 +1,10 @@
+import { Encoder, Parser, useStorageBackedState } from './StorageBackedState';
+
+export function useLocalBackedState<T>(
+  defaultValue: T,
+  key: string,
+  encode: Encoder<T>,
+  parse: Parser<T>,
+) {
+    return useStorageBackedState(window.localStorage, defaultValue, key, encode, parse)
+  }
```

Comment:
Just a small formatting nitpick! You can click "commit suggestion" to automatically commit to the PR.
```suggestion
) {
  return useStorageBackedState(window.localStorage, defaultValue, key, encode, parse);
}
```